### PR TITLE
Add GPL3 license

### PR DESCRIPTION
**Pull request by jletey**
This is a really useful license!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -372,6 +374,713 @@ As an alias:
...
+license-up gpl3
+```
+
+Here is a working example of it.
+
+```Bash
+license-up gpl3
+```
+
+Will create a file `LICENSE` with following content:
```
Comment (nikivdev) [inline README.md:393]: I think we shouldn't put the entire GNU license in README. It's way too long. Perhaps we can truncate to only show the top and mention it?

**Review by nikivdev — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jletey) [reply README.md:393]: There's really two options here ... we can include all of the text, but have it hidden like this:

<details><summary>Some hidden text</summary>
All of the hidden content
</details>

Or we can just include the [header](https://oss.ninja/gpl-3.0-header) of the license

Comment (nikivdev) [reply README.md:393]: The hidden text is nice. Let's do that for all other licences too. I think it would be a good change.

Comment (jletey) [reply README.md:393]: Sounds good ... I'll do it for this one, and then create a new pull request for all of the other ones

**Pull request merged**